src/utils/calculateDriverSleepStats.js

```
@@ -9,7 +9,8 @@ export function calculateDriverSleepStats(sleepList, driverHash) {
   const hourCount = {};
   sleepList.forEach((item) => {
     if (item.driverHash !== driverHash) return;
-    const hour = new Date(item.detectedTime).getUTCHours();
+    const isoString = item.detectedTime;
+    const hour = parseInt(isoString.split('T')[1].split(':')[0], 10);
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)